File: app/src/main/java/com/sd/mvc/intercept_video_link/utils/JsonHelper.java
package com.sd.mvc.intercept_video_link.utils;

import com.google.gson.Gson;

public class JsonHelper {
    private static Gson gson = new Gson();

    public static String jsonToString(Object obj) {
        return gson.toJson(obj);
    }

    public static Object stringToJson(String json, Class clazz) {
        return gson.fromJson(json, clazz);
    }
}

File: app/src/main/java/com/sd/mvc/intercept_video_link/utils/JsonHelper.kt
package com.sd.mvc.intercept_video_link.utils

import com.google.gson.Gson
import com.google.gson.reflect.TypeToken

import java.lang.reflect.Type
import java.util.HashMap

object JsonHelper {
    private val gson = Gson()

    fun jsonToString(obj: Any): String {
        return gson.toJson(obj)
    }

    fun stringToJson(json: String, clazz: Class<*>): Any {
        return gson.fromJson<Any>(json, clazz)
    }
}
